.gitignore

```diff
@@ -11,6 +11,17 @@ obj
 /config.log
 /config.status
 /configure
+/configure~
 /ltmain.sh
 /config.guess
 /config.sub
+/missing
+/Makefile
+/Makefile.in
+/NEWS
```

Comment:
NEWS, AUTHORS, COPYING, ChangeLog shouldnt be in .gitignore.

Reply:
Fixed